### PR TITLE
Remove a potentially pre-existing server.pid in docker-compose.yml

### PR DESCRIPTION
**Pull request by raoulkramer**
This command will tigger the script in the entrypoint.sh to remove a potentially pre-existing server.pid for Rails. With this change the ops does not have to manually [rm tmp/pids/server.pid] remove the server.pid file when the container is restarted.

### CODE REVIEW

File: docker-compose.yml
Comment (javierm) [reply docker-compose.yml:22]: > I tested this command and it doesn't seem to give any problems. Maybe you should check that too in your dockerless setup.

It runs without problems, but then we'd have a server running with no `server.pid` file, and who knows what would happen.

> My question: does your plain (dockerless) rails setup use the entrypoint.sh file?

Not at all; just a regular Rails+puma application.

> When I run the docker compose up command (with rm server.pid command), it will indicate that the containers are already running and not remove the running PID server.

Ah, alright. Then I'm fine with this change :+1:.

I've approved this pull request, but since you've mentioned this one might not be necessary, I'll wait for your confirmation before merging it. Shall we merge this one?

**Review by javierm — APPROVED** [review]

File: docker-compose.yml
Comment (raoulkramer) [reply docker-compose.yml:22]: > I've approved this pull request, but since you've mentioned this one might not be necessary, I'll wait for your confirmation before merging it. Shall we merge this one?

Yes that would be great! :+1: 

_Future enhancements, later on, could be a additional production specific docker compose file_ `production.yml` https://docs.docker.com/compose/production/

Comment (javierm) [reply docker-compose.yml:22]: So, then, shall we close this one in favor of #4684? :thinking:

Comment (raoulkramer) [reply docker-compose.yml:22]: Yes, I think #4684 is neater. Sorry for the confusion. :pretzel:

**Comment by javierm** [comment]
Closing in favor of #4684 :ok_hand:.

**Pull request closed without merging**
